internal/image/fetcher.go

```diff
@@ -28,6 +28,14 @@ type Fetcher struct {
 	logger logging.Logger
 }
 
+type FetchOptions struct {
+	//TODO: would love to reverse (Daemon bool) to (Remote bool)
+	//      so that "false" becomes the default, when omitted
```

Comment:
We would rather not commit TODOs without an issue created and being referenced. This particular TODO seems a bit superfluous, not sure it's worth pursuing outside of this change. I'd be open to resolving this TODO as part of this PR or removing the TODO altogether. My 2c.